MathPortal/MathPortal/Source/API/ParseObject.swift

```diff
@@ -45,6 +45,11 @@ class ParseObject {
         return PFQuery(className: entityName)
     }
     
+    static func pfObjectId(objectId: String?) -> PFObject? {
```

Comment:
Probably a better name would be `withId(_ id: String?) -> PFObject`. This way it is then for instance `let myObjet = MyObject.withId(someId)`